src/sasconf.h

```diff
@@ -14,7 +14,11 @@
 
 #include <sys/param.h>
 
-#define __BIGREGION__ 1
+#if defined(__BIGREGION__) && __BIGREGION__ == 0
+# undef __BIGREGION__
+#else
+# define __BIGREGION__ 1
+#endif
```

Comment:
If you make this change __BIGREGION__ will always be disabled unless someone manually specifies that at build time.
Isn't it better to enable or disable it automatically depending on the host (i.e. configure --host) value?